tests/unit_tests/test_routes.py

```diff
@@ -21,6 +21,14 @@ def test_checker_ok(self):
         self.assertEqual(response.content_type, "application/json; charset=utf-8")
         self.assertEqual(response.json, {'success': True, 'message': 'OK'})
 
+    def test_create_shortlink_options(self):
+        response = self.app.options(
+            url_for('create_shortlink'), headers={"Origin": "map.geo.admin.ch"}
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content_type, "application/json; charset=utf-8")
+        self.assertEqual(response.data.decode("utf-8"), '')
```

Comment:
Would be very good to test if the `Allow` header is present as it is the goal of OPTIONS.